content/_data/plugins.json

```diff
@@ -643,6 +643,14 @@
           "link": "https://picklejs.com",
           "keywords": ["cucumber", "collection", "actions", "commands"],
           "badge": "community"
+        }, 
+        {
+          "name": "Pact Cypress Adapter",
+          "description": "Simple commands that generates pact from Cypress for pact test",
+          "link": "https://github.com/pactflow/pact-cypress-adapter",
+          "keywords": ["pact", "contract testing", "commands"],
+          "badge": "official"
```

Comment:
```suggestion
          "badge": "community"
```

Reply:
Good shout @admah 

We've also address a description change we wanted to make, in the following PR https://github.com/cypress-io/cypress-documentation/pull/4477

@B3nnyL is based in Aus, so will probably be 💤  :)

Reply:
Happy for her to update the comments and get the full PR glory though 🙌🏾  🚀